packages/edit-site/src/hooks/use-edited-entity-from-params.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreDataStore } from '@wordpress/core-data';
+
+const postTypesWithoutParentTemplate = [
+	'wp_template',
+	'wp_template_part',
+	'wp_block',
+	'wp_navigation',
+];
+
+export default function useEditedEntityForParams( params ) {
```

Comment:
This is the hook that can be used to resolve what postType and postId to use to render an editor frame for the given (postType, postId) tuple. If none is provided we fallback to the home page.